package.json

```diff
@@ -109,9 +109,9 @@
     "@chbphone55/classnames": "2.0.0",
     "@lingui/core": "4.7.0",
     "@warp-ds/core": "1.0.2",
-    "@warp-ds/css": "1.8.0",
+    "@warp-ds/css": "1.8.2-next.2",
```

Comment:
https://github.com/warp-ds/css/pull/167 ?

Reply:
Updated to 1.8.2 in [6caf68b](https://github.com/warp-ds/react/pull/206/commits/6caf68b73eeb19651e9bf0748cc4b684b7e2a314)
I wanted the preview to include the new classes for the a11y team to be able to test Pagination.